lib/overcommit/configuration.rb

```diff
@@ -37,6 +39,22 @@ def plugin_directory
       File.join(Overcommit::Utils.repo_root, @hash['plugin_directory'] || '.git-hooks')
     end
 
+    # Returns absolute path to directoy for the history of signatures
+    # @return [String] - Absolute path to the signature directory
+    def signature_directory
+      File.join(Overcommit::Utils.repo_root,
+                '.git',
+                @hash['signature_directory'] || 'overcommit-signatures')
```

Comment:
Can you elaborate on why allowing this to be customized is useful? I'm fearful of an overzealous user changing this to a directory of files which are tracked by their Git repository because it can be "helpful" to others in storing previous signatures. Of course this is problematic since now an attacker can modify said files.

I anticipate this is a highly-unlikely scenario, but I would prefer to stick to the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle unless there's a solid reason to allow this to be customized. Curious to hear your thoughts here.